things/api/things/http/transport.go

```diff
@@ -479,6 +408,34 @@ func decodeCreateConnections(_ context.Context, r *http.Request) (interface{}, e
 	return req, nil
 }
 
+func decodeListThingsGroupRequest(_ context.Context, r *http.Request) (interface{}, error) {
+	o, err := readUintQuery(r, offsetKey, defOffset)
+	if err != nil {
+		return nil, err
+	}
+
+	l, err := readUintQuery(r, limitKey, defLimit)
+	if err != nil {
+		return nil, err
+	}
+
+	m, err := readMetadataQuery(r, metadataKey)
+	if err != nil {
+		return nil, err
+	}
+
+	req := listThingsGroupReq{
+		token:   r.Header.Get("Authorization"),
+		groupID: bone.GetValue(r, "groupId"),
```

Comment:
String constant. Also, is it "groupId" or "groupID"?

Reply:
@nmarcetic @dusanb94 @manuio what do we do with these string constants? We leave them here or @mteodor declares a constant?

Reply:
It would be better but we should align in other services... What is more important is to fix `groupID`

Reply:
this is aligned to other services